components/datetime/tests/datetime.rs

```diff
@@ -99,7 +99,7 @@ fn test_fixture(fixture_name: &str, file: &str) {
             None => format!("\n  file: {fixture_name}.json\n"),
         };
         for (locale, output_value) in fx.output.values {
-            let options = options_base.clone();
+            let options = options_base;
```

Comment:
> nitpick (optional)

Since this is `Copy` now, there's really no need for this whole line. 

We could just initialize `options_base` as `options` instead, and let `Copy` do its thing in the loop.

I assume the original author called it `options_base` because they didn't want to use shadowing (`let options = options.clone()`), or put an explicit `options.clone()` in each assertion. 

But it's just test code, so if you don't want to request yet another review, I don't think it's an issue.

Reply:
fixed, thank you